### PR TITLE
Add Cryptorank envs

### PR DESCRIPTION
**Pull request by nikitosing**
https://github.com/blockscout/blockscout/pull/10550

### CODE REVIEW

File: setup-and-run-blockscout/env-variables/backend-env-variables.md [hunk 1]
```diff
@@ -231,6 +231,14 @@ _**Note**_**: Before release 5.1.2, all environment variables of time format sup
...
+| `EXCHANGE_RATES_CRYPTORANK_SECONDARY_COIN_ID`        | Sets Cryptorank coin ID for secondary coin market chart. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |
+| `EXCHANGE_RATES_CRYPTORANK_PLATFORM_ID`              | Sets Cryptorank platform ID. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |
+| `EXCHANGE_RATES_CRYPTORANK_BASE_URL`                 | If set, overrides the Cryptorank API url. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>coin_gecko</code><br>Applications: API, Indexer</p>            |
```
Comment (fedor-ivn) [inline setup-and-run-blockscout/env-variables/backend-env-variables.md]: > <code>coin_gecko</code>

I guess there should be `https://api.cryptorank.io/v1/`

Comment (fedor-ivn) [reply setup-and-run-blockscout/env-variables/backend-env-variables.md]: Also check the entry for `EXCHANGE_RATES_CRYPTORANK_API_KEY`

File: setup-and-run-blockscout/env-variables/backend-env-variables.md [hunk 2]
```diff
@@ -231,6 +231,14 @@ _**Note**_**: Before release 5.1.2, all environment variables of time format sup
...
+| `EXCHANGE_RATES_CRYPTORANK_COIN_ID`                  | Sets Cryptorank coin ID. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: (empty)<br>Applications: API, Indexer</p>            |
+| `EXCHANGE_RATES_CRYPTORANK_LIMIT`                    | Sets Cryptorank "batch size" (limit on amount of prices returned for one request) for for tokens price fetching. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>1000</code><br>Applications: API, Indexer</p>            |
```
Comment (fedor-ivn) [inline setup-and-run-blockscout/env-variables/backend-env-variables.md]: I wanted to notice repetition of _for_, but also refactored description. You may leave the one you wrote if this one seems to be less clear.

```suggestion
| `EXCHANGE_RATES_CRYPTORANK_LIMIT`                    |Sets the maximum number of token prices returned in a single request. Implemented in [#10550](https://github.com/blockscout/blockscout/pull/10550).                                                                                                                               | <p>Version: master<br>Default: <code>1000</code><br>Applications: API, Indexer</p>            |
```

**Review by fedor-ivn — CHANGES_REQUESTED** [review]

File: setup-and-run-blockscout/env-variables/backend-env-variables.md (hunk 2 above)
Comment (nikitosing) [reply setup-and-run-blockscout/env-variables/backend-env-variables.md]: Nice catch! Fixed

File: setup-and-run-blockscout/env-variables/backend-env-variables.md (hunk 1 above)
Comment (nikitosing) [reply setup-and-run-blockscout/env-variables/backend-env-variables.md]: Nice catch! Fixed

**Review by fedor-ivn — APPROVED** [review]

**Pull request merged**